src/z3c/dependencychecker/dependencychecker.py

```diff
@@ -84,6 +84,57 @@
 ['\"]        # Single or double quote.
 """, re.VERBOSE)
 
+FTI_BEHAVIOR_PATTERN = re.compile(r"""
+\s           # Whitespace.
+<element     #
+\s+          #
+value=       #
+['\"]        # Single or double quote.
+(?P<import>  # Start of 'import' variable.
+\S+          # Non-whitespace string.
+)            # End of 'import' variable.
+['\"]        # Single or double quote.
+""", re.VERBOSE)
+
+FTI_SCHEMA_PATTERN = re.compile(r"""
+\s           # Whitespace.
+name=        #
+['\"]        # Single or double quote.
+schema       #
+['\"]        # Single or double quote.
+>            # End of opening tag
+(?P<import>  # Start of 'import' variable.
+\S+          # Non-whitespace string.
```

Comment:
You can pretty much safely asume that the package is in a sane state. Once you start using a dependencychecker to clean up your dependency graph, you're pretty much guaranteed to have well-behaved packages.

So it doesn't need to be made more elaborate or fail-safe than now, I think.